cmd/modulectl/create/scaffold/scaffold.go

```diff
@@ -0,0 +1,95 @@
+allowing for a tailored scaffolding experience according to the specific needs of the module being created.
+
+The command generates or uses the following files:
+ - Module Config:
+	Enabled: Always
+	Adjustable with flag: --module-config=VALUE
+	Generated when: The file doesn't exist or the --overwrite=true flag is provided
+	Default file name: scaffold-module-config.yaml
+ - Manifest:
+	Enabled: Always
+	Adjustable with flag: --gen-manifest=VALUE
+	Generated when: The file doesn't exist. If the file exists, it's name is used in the generated module configuration file
+	Default file name: manifest.yaml
+ - Default CR(s):
+	Enabled: When the flag --gen-default-cr is provided with or without value
+	Adjustable with flag: --gen-default-cr[=VALUE], if provided without an explicit VALUE, the default value is used
+	Generated when: The file doesn't exist. If the file exists, it's name is used in the generated module configuration file
+	Default file name: default-cr.yaml
+ - Security Scanners Config:
+	Enabled: When the flag --gen-security-config is provided with or without value
+	Adjustable with flag: --gen-security-config[=VALUE], if provided without an explicit VALUE, the default value is used
+	Generated when: The file doesn't exist. If the file exists, it's name is used in the generated module configuration file
+	Default file name: sec-scanners-config.yaml
+
+**NOTE:**: To protect the user from accidental file overwrites, this command by default doesn't overwrite any files.
+Only the module config file may be force-overwritten when the --overwrite=true flag is used.
+
+You can specify the required fields of the module config using the following CLI flags:
+--module-name=NAME
+--module-version=VERSION
+--module-channel=CHANNEL
+
+**NOTE:**: If the required fields aren't provided, the defaults are applied and the module-config.yaml is not ready to be used. You must manually edit the file to make it usable.
+Also, edit the sec-scanners-config.yaml to be able to use it.
+`,
+		Example: `Generate a minimal scaffold for a module - only a blank manifest file and module config file is generated using defaults
+                kyma alpha create scaffold
+Generate a scaffold providing required values explicitly
+                kyma alpha create scaffold --module-name="kyma-project.io/module/testmodule" --module-version="0.1.1" --module-channel=fast
+Generate a scaffold with a manifest file, default CR and security-scanners config for a module
+                kyma alpha create scaffold --gen-default-cr --gen-security-config
+Generate a scaffold with a manifest file, default CR and security-scanners config for a module, overriding default values
+                kyma alpha create scaffold --gen-manifest="my-manifest.yaml" --gen-default-cr="my-cr.yaml" --gen-security-config="my-seccfg.yaml"
+
+`,
+		Args: cobra.ExactArgs(0),
+		RunE: scaffold.RunE,
+	}
+
+	cmd.Flags().StringVar(&scaffold.ModuleName, "module-name", "kyma-project.io/module/mymodule",
```

Comment:
There is inconsistency in terms of that flag names are sometimes consts sometimes direct values, like here.
Let's extract all to consts so we may unit test them as well.